README.md

```diff
@@ -117,8 +117,7 @@ dirctl build [options] <path-to-agent>
 Options:
 - `--name` : Name of the agent.
 - `--version` : Version of the agent.
-- `--artifact-url` : URL of the agent artifact.
-- `--artifact-type` : Type of the agent artifact.
+- `--artifact` : Artifacts to set for the agent. Each artifact should be in the format `type:url`. Example usage: `--artifact type1:url1 --artifact type2:url2`.
```

Comment:
Can we just have a description of allowed types, for example:
```suggestion
- `--artifact` : Artifacts to set for the agent. Each artifact should be in the format `type:url`. Example usage: `--artifact type1:url1 --artifact type2:url2`. Supported types: `docker-image`, `python-package`.
```

Reply:
neat 👍  will just add `helm-chart`

Reply:
we also need one more locator type to be defined `source` for pointing to the source code

Reply:
```proto
enum LocatorType {
  LOCATOR_TYPE_UNSPECIFIED = 0;
  LOCATOR_TYPE_HELM_CHART = 1;
  LOCATOR_TYPE_DOCKER_IMAGE = 2;
  LOCATOR_TYPE_PYTHON_PACKAGE = 3;
  LOCATOR_TYPE_SOURCE_CODE = 4;
}
```

also update the mapping here https://github.com/agntcy/dir/blob/1419ecc3ef4f497a59bd77ea95575076a6ee0412/api/core/v1alpha1/locator.go#L6

Reply:
should we also add `binary`?